gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/HiveRegistrationUnit.java

```diff
@@ -418,6 +421,11 @@ public T withTableName(String tableName) {
       return (T) this;
     }
 
+    public T withRegisterSchema(boolean registerSchema) {
```

Comment:
What's the usage of this field?

Reply:
This is the builder method, so that our gobblin-hive-registration can set weather the partition needs to register schema.

Reply:
Put it this way, the getHivePartition's actual implementation is in gobblin-hive-registration MP, which is out of the open source code base.

Reply:
Fair enough.